src/commonMain/kotlin/exchange.dydx.abacus/state/manager/StateManagerAdaptor.kt

```diff
@@ -432,9 +436,11 @@ open class StateManagerAdaptor(
             }
             if (sourceAddress != null) {
                 screenSourceAddress()
+                sourceAddress?.let { complianceScreen(it) }
```

Comment:
Nit: Since there is already a sourceAddress != null check, shouldn't need to do a let.  Can simply call complianceScreen(sourceAddress)